### ISSUE TITLE
Hang/Freeze on startup: caused by NahimicOSD.dll

### ISSUE THREAD

**Issue by ThirteenAG** [ThirteenAG/GTAIV.EFLC.FusionFix#356]
### Game version this happens in

v1.2.0.59

### What area of the game

If your game freezes on startup, it's caused by NahimicOSD.dll. Uninstalling the software that injects it will help, however I'm trying to find a way to disable its injection via Fusion Fix. Reply in this thread if you want to help.

### What should happen

It shouldn't crash.

### CrashDumps and logs

![image](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/assets/4904157/a2476eaf-cf89-4d07-84fb-4b9c286e034e)

```
>	ntdll.dll!_NtWaitForMultipleObjects@20()	
 	KERNELBASE.dll!_WaitForMultipleObjectsEx@20()	
 	KERNELBASE.dll!_WaitForMultipleObjects@16()	
 	d3d9.dll!ThreadAffinitizedWorkerThread<enum ProxyWindowCommands>::PostCommand()	
 	d3d9.dll!ProxyWindow::CreateProxyWindow(struct HWND__ *,struct tagRECT const &,enum ProxyWindowBehavior)	
 	d3d9.dll!CSwapChain::InitBindResources(bool,struct _SECURITY_DESCRIPTOR *)	
 	d3d9.dll!CSwapChain::CreateWindowed(unsigned int,unsigned int,enum _D3DFORMAT,unsigned int,enum _D3DMULTISAMPLE_TYPE,unsigned long,int,int,int,int,int,int)	
 	d3d9.dll!CSwapChain::Reset(struct _D3DPRESENT_PARAMETERS_ *,struct D3DDISPLAYMODEEX const *,int,int)	
 	d3d9.dll!CSwapChain::Init(struct _D3DPRESENT_PARAMETERS_ *,struct D3DDISPLAYMODEEX const *,long *,int,int)	
 	d3d9.dll!CBaseDevice::Init()	
 	d3d9.dll!CEnum::CreateDeviceImpl(unsigned int,enum _D3DDEVTYPE,struct HWND__ *,unsigned long,struct _D3DPRESENT_PARAMETERS_ *,struct D3DDISPLAYMODEEX const *,struct IDirect3DDevice9Ex * *,struct _D3D9ON12_ARGS *)	
 	d3d9.dll!CEnum::CreateDevice()	
 	NahimicOSD.dll!77ef7e27()	Unknown	Non-user code. Cannot find or open the PDB file.
```

### Checklist

- [X] Test in the [latest git build](https://ci.appveyor.com/api/projects/ThirteenAG/gtaiv-eflc-fusionfix/artifacts/GTAIV.EFLC.FusionFix.zip) in case it's already fixed.
- [X] [Search for other reports](https://github.com/ThirteenAG/GTAIV.EFLC.FusionFix/search?q=my+issue&type=issues) of the same issue.
- [X] Include logs or screenshots of issue.

**Comment by yxngicyx** [comment]
Hello there, i am the one that pointed out the issue, uninstalling nahimic prevents the game from freezing, but i would get crashes 1 hour into the game [doing anything; missions, free roaming] with the AMD Adrenaline software overlay on, having this disabled prevents the crash, but i notice other weird behaviors with RTSS & Steam overlay, when having rtss osd & steam overlay on, when using steam overlay [shift + tab] and then going back into the game, my cursor stays on the screen.

**Comment by RecklessGlue540** [comment]
> Hello there, i am the one that pointed out the issue, uninstalling nahimic prevents the game from freezing, but i would get crashes 1 hour into the game [doing anything; missions, free roaming] with the AMD Adrenaline software overlay on, having this disabled prevents the crash, but i notice other weird behaviors with RTSS & Steam overlay, when having rtss osd & steam overlay on, when using steam overlay [shift + tab] and then going back into the game, my cursor stays on the screen.

Attach a Crash Dump for each problem.

**Comment by ThirteenAG** [comment]
> Hello there, i am the one that pointed out the issue, uninstalling nahimic prevents the game from freezing, but i would get crashes 1 hour into the game [doing anything; missions, free roaming] with the AMD Adrenaline software overlay on, having this disabled prevents the crash, but i notice other weird behaviors with RTSS & Steam overlay, when having rtss osd & steam overlay on, when using steam overlay [shift + tab] and then going back into the game, my cursor stays on the screen.

Can you install it back and post your discord so I can contact you?

**Comment by yxngicyx** [comment]
> > Hello there, i am the one that pointed out the issue, uninstalling nahimic prevents the game from freezing, but i would get crashes 1 hour into the game [doing anything; missions, free roaming] with the AMD Adrenaline software overlay on, having this disabled prevents the crash, but i notice other weird behaviors with RTSS & Steam overlay, when having rtss osd & steam overlay on, when using steam overlay [shift + tab] and then going back into the game, my cursor stays on the screen.
> 
> Can you install it back and post your discord so I can contact you?

have it installed again, here's my discord: sxulxy

**Comment by ThirteenAG** [comment]
Resolved in 2.3.1.
